prediction_market_agent/db/pinecone_handler.py

```diff
@@ -0,0 +1,95 @@
+import base64
+import sys
+import typing as t
+from datetime import datetime
+from typing import Optional
+
+from langchain_openai import OpenAIEmbeddings
+from langchain_pinecone import PineconeVectorStore
+from loguru import logger
+from pinecone import Pinecone
+from prediction_market_agent_tooling.markets.agent_market import FilterBy, SortBy
+from prediction_market_agent_tooling.markets.omen.omen_subgraph_handler import (
+    OmenSubgraphHandler,
+)
+from tqdm import tqdm
+
+from prediction_market_agent.agents.think_thoroughly_agent.models import (
+    PineconeMetadata,
+)
+from prediction_market_agent.utils import APIKeys
+
+INDEX_NAME = "omen-markets"
+T = t.TypeVar("T")
+
+
+class PineconeHandler:
+    def __init__(self) -> None:
+        k = APIKeys()
```

Comment:
```suggestion
        keys = APIKeys()
```

Single-char variable names should almost never be used please 🙏